pyv/transpiler.py

```diff
@@ -222,8 +222,15 @@ def visit_Call(self, node: ast.Call) -> str:
 
         vargs: List[str] = []
 
-        if node.args:
-            vargs.extend(map(self.visit, node.args))
```

Comment:
I noticed that pyv doesn't have `visit_arg`. Wouldn't it be better to add one and use logic similar to pyrs to handle mutable args?

Reply:
@div72 - pinging this thread to make sure you saw my comment.

Reply:
Correct me if I am mistaken but call args are not ast.arg nodes, no? And in function definitions, making all unknown types generic T is not the best idea as there's no guarantee that the arguments have the same generic type. That's the reason I rewrited it into `function_signature` function.

Reply:
I guess I can make it back into a visit_arg with holding generic count as an int in the class but that'll get messy when someone uses generics from python side.

Reply:
correct. There is no ast.arg node on Call. Only on FunctionDef.

But if you look at https://github.com/adsharma/py2many/blob/main/pycpp/transpiler.py#L185-L188

The case where different arguments have different generic types can be handled ok.

Having the same general structure as pycpp and pyrs helps promote common code to base class sometimes.

Reply:
Fixed.